lib/msf/core/modules/metadata/search.rb

```diff
@@ -52,8 +55,13 @@ def is_match(search_string, module_metadata)
         res[f][0] << v
       end
     end
+    res
+  end
 
-    k = res
+  def is_match(params, module_metadata)
+    return false if params.empty?
+
+    k = params
```

Comment:
I know you didn't originally write this, but would you be willing to give these single character variables more readable names? "No" is a totally valid answer, but wanted to present it since we were already working here.

Reply:
Yeah, for sure. I wanted to change as little as possible so as not to introduce mistakes, but I'm all for readability

Reply:
Awesome, thank you so much!!

Reply:
Names are now verbose